docs/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ Entries inside each section should be ordered by type:
 ## CLI
 !-->
 
+# unreleased - YYYY-MM-DD
+
+## Python API
+
+* [Added] `quilt3.admin.tabulator`: `get_open_query` and `set_open_query` functions for managing Tabulator open query mode ([#4260](https://github.com/quiltdata/quilt/pull/4260))
```

Comment:
I'm used to the function references in Python docs being rendered with brackets:

```suggestion
* [Added] `quilt3.admin.tabulator`: `get_open_query()` and `set_open_query()` functions for managing Tabulator open query mode ([#4260](https://github.com/quiltdata/quilt/pull/4260))
```